src/assets/scss/navbar.scss

```diff
@@ -56,7 +56,7 @@
         &-text {
           display: none;
           margin-left: 8px;
-          color: $fontColor;
```

Comment:
How about update the theme file?
`gophercon2020/src/assets/scss/theme/dark.scss`
`gophercon2020/src/assets/scss/theme/light.scss`
to
**@include navbar($blue-5, $white, $dark-1);**

What do you think?